src/app_dir.rs

```diff
@@ -87,13 +92,38 @@ pub fn copy_env_file_if_not_exists() {
         app_user_data_dir_path().expect("Failed to determine application data directory");
     let env_file_in_app_dir = app_data_dir.join(".env".to_string());
     if !env_file_in_app_dir.exists() || !env_file_in_app_dir.is_file() {
+        // Get the directory where the executable is located
+        let exe_dir = std::env::current_exe()
```

Comment:
I think it would be better to embed the default .env file inside the binary.  Looks like `include_bytes!` can do the job, but I didn't test this.